src/site/stages/build/process-cms-exports/index.js

```diff
@@ -193,6 +215,7 @@ const entityAssemblerFactory = contentDir => {
         parentFieldName,
         contentDir,
         assembleEntityTree,
+        transformUnpublished,
```

Comment:
It looks like we're passing this extra boolean argument to `transformEntity` defined in `src/site/stages/build/process-cms-exports/transform.js`. Do you want to add an arg there?